maps-compose/src/main/java/com/google/maps/android/compose/RememberComposeBitmapDescriptor.kt

```diff
@@ -39,17 +41,23 @@ private fun renderComposableToBitmapDescriptor(
     val composeView =
         ComposeView(parent.context)
             .apply {
+                layoutParams = ViewGroup.LayoutParams(
+                    ViewGroup.LayoutParams.WRAP_CONTENT,
+                    ViewGroup.LayoutParams.WRAP_CONTENT,
+                )
                 setParentCompositionContext(compositionContext)
                 setContent(content)
             }
             .also(parent::addView)
 
     composeView.draw(fakeCanvas)
 
-    composeView.measure(
-        View.MeasureSpec.makeMeasureSpec(parent.width, View.MeasureSpec.AT_MOST),
-        View.MeasureSpec.makeMeasureSpec(parent.height, View.MeasureSpec.AT_MOST),
-    )
+    composeView.measure(measureSpec, measureSpec)
+
+    if (composeView.measuredWidth == 0 || composeView.measuredHeight == 0) {
+        throw IllegalStateException("The ComposeView was measured to have a width or height of " +
+                "zero. Make sure the parent and content have a non-zero size.")
```

Comment:
According to your testing, "parent" size should no longer matter, right?

Reply:
Yeah, right. I changed that.